shortlivedpool.go

```diff
@@ -0,0 +1,17 @@
+// +build shortlivedpool
+
+package bytebufferpool
+
+import (
+	"log"
+
+	"github.com/gallir/shortlivedpool"
+)
+
+type actualPool struct {
+	pool shortlivedpool.Pool
+}
+
+func init() {
+	log.Println("Using github.com/gallir/shortlivedpool instead of sync.pool")
```

Comment:
I wouldn't do this. It's super annoying if there is no way for people to disable this. Seeing as it's in `init`, packages that use this package don't have the possibility to call `log.SetOutput` before this.